### PR TITLE
Patch of v0.1.0 to v0.1.1 of WeaveFlux integration. 

### PR DESCRIPTION
**Pull request by TheJaySmith**
Per our meeting, I did some minor upgrades (largely in README.md) to make it more intuitive.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1338)

### CODE REVIEW

**Comment by pdmack** [comment]
@TheJaySmith can you provide a more descriptive title please?

**Comment by TheJaySmith** [comment]
I changed the title. /retest

**Comment by pdmack** [comment]
#1348 
`"Master version \"1.9.7-gke.3\" is unsupported."`

Can you try rebasing your changes please? I _think_ some fixes went in for this.

**Comment by pdmack** [comment]
```
Manifest expansion encountered the following errors: while scanning a simple key                                                                                                    
     in "<unicode string>", line 46, column 1:                                                                                                                                         
       <<<<<<< HEAD                                                                                                                                                                    
       ^                                                                                                                                                                               
   could not find expected ':'                                                                                                                                                         
     in "<unicode string>", line 47, column 28:                                                                                                                                        
             initialClusterVersion: 1.9.7-gke.5                                                                                                                                        
                                  ^ Resource: config
```

Looks like the rebase/merge needs some tidying up.

File: kubeflow/weaveflux/prototypes/weaveflux.jsonnet [hunk 1]
```diff
@@ -1,10 +1,10 @@
-// @apiVersion 0.1
+// @apiVersion 0.1.1
 // @name io.ksonnet.pkg.weaveflux
 // @description Weaveworks Flux integration with Kubeflow
 // @shortDescription A Flux meets Kubeflow
 // @param name string Name to give to each of the components
 // @optionalParam giturl string null Your default git URL.
-// @optionalParam serviceType string ClusterIP The service type for WeaveFlux.
+// @optionalParam serviceType string LoadBalancer The service type for WeaveFlux.
```
Comment (jlewi) [inline kubeflow/weaveflux/prototypes/weaveflux.jsonnet]: Please don't do this. Creating a LoadBalancer by  default is opening up an external port. We do not want our default configs to be opening up external access by default. 
Our default operation should always be secure; even if that means telling people to do inconvenient things like use kubectl port-forward.

On GCP we should really try to make this secure using IAP.

Comment (TheJaySmith) [reply kubeflow/weaveflux/prototypes/weaveflux.jsonnet]: How would you feel about putting it behind ambassador or envoy or something similar? I feel like that's a more automated yet secure method. making a user port-forward isn't the end of the world but I would like to automate it and make it secure.

Comment (TheJaySmith) [reply kubeflow/weaveflux/prototypes/weaveflux.jsonnet]: Went ahead and did the NodePort. I have a long-term plan of using Ambassador and will include it in the v0.2 of WeaveFlux with the CLI tool.

**Pull request closed without merging**